custom_components/easee/services.yaml

```diff
@@ -117,7 +117,7 @@ set_circuit_max_current:
       example: 10
 
 set_charger_circuit_dynamic_current:
-  description: Set circuit dynamic current
+  description: Set circuit dynamic current (current available to share by several chargers on a circuit, to be set on master charger).
```

Comment:
The last part about the master charger is unclear to me. The API we call is not actually associated with the charger, so why would this be the case?